### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by apdavison**
[ci skip]

### CODE REVIEW

File: doc/source/io.rst [hunk 1]
```diff
@@ -108,84 +108,81 @@ All IOs have a read() method that returns a list of :class:`Block` objects (repr
...
-    shape1 = seg.spiketrains[0].shape  # total shape
+    seg = reader.read_segment(time_slice=None)
 
-    
-This read only the first 5 seconds::
+This reads only the first 5 seconds::
 
-    seg = reader.load_segment(time_slice=(0*pq.s, 5.*pq.s))
-    shape0 = seg.analogsignals[0].shape # more small shape
-    shape1 = seg.spiketrains[0].shape  # more small shape
```
Comment (samuelgarcia) [inline doc/source/io.rst:130]: shape0/shape1 : was this too obvious or is it too much detail ?

**Comment by samuelgarcia** [comment]
Thank you.
I am sorry for all my spelling errors.

File: doc/source/io.rst (hunk 1 above)
Comment (apdavison) [reply doc/source/io.rst:130]: I thought it was too much detail.

But I don't feel strongly about it, so no problem if you want to change it back.

Comment (samuelgarcia) [reply doc/source/io.rst:130]: No worry it is OK like that.

**Pull request closed without merging**